urlang/main.rkt

```diff
@@ -1473,12 +1473,26 @@
 
 (define (parse-object o)
   (debug (list 'parse-object (syntax->datum o)))
+  (define (duplicates? pns)
+      (remove-duplicates (map syntax-e (filter (λ(item) 
+        (>
+          (length
+            (filter (λ(other) (free-identifier=? item other)) pns))
+        1))
+      pns)
+  )))
+
+
   (with-output-language (Lur Expr)
```

Comment:
This looks like the right idea. 

I suggest using `check-duplicates` from `racket/list` with `free-identifier=?` as "same" [1].
It confuses me slightly that the return value of `duplicates?` is a list. 
Usually functions that has a named that ends in `?` return `#f` or "a true value". 
Note that `check-duplicates` returns `#f` to mean that no duplicates were found.

[1] https://docs.racket-lang.org/reference/pairs.html#%28def._%28%28lib._racket%2Flist..rkt%29._check-duplicates%29%29

Reply:
I must have clicked in the wrong spot. The comment above is with regards to:

```
(define (duplicates? pns)
      (remove-duplicates (map syntax-e (filter (λ(item) 
        (>
          (length
            (filter (λ(other) (free-identifier=? item other)) pns))
        1))
      pns)
  )))
  ```